### ISSUE TITLE
Showing new listings for Wednesday, 5 February 2025

### ISSUE THREAD

**Issue by terrryahn78** [terrryahn78/daily_arxiv_noti#82]
## Self-induced Josephson oscillations and self-trapping in a supersolid dipolar quantum gas
- **Keywords:** ['Josephson junction array', 'Josephson junction array']
 - **Authors:** Beatrice Donelli, Nicolò Antolini, Giulio Biagioni, Marco Fattori, Andrea Fioretti, Carlo Gabbanini, Massimo Inguscio, Luca Tanzi, Giovanni Modugno, Augusto Smerzi, Luca Pezzè
 - **Subjects:** Subjects:
Quantum Gases (cond-mat.quant-gas); Quantum Physics (quant-ph)
 - **Arxiv link:** https://arxiv.org/abs/arXiv:2501.17142
 - **Pdf link:** https://arxiv.org/pdf/arXiv:2501.17142
 - **Abstract**
 The Josephson effect characterizes superfluids and superconductors separated by a weak link, the so-called Josephson junction. A recent experiment has shown that Josephson oscillations can be observed also in a supersolid, where the weak link is not due to an external barrier, but is self-induced by interparticle interactions. Here we show theoretically that supersolids -- despite their self-induced character -- feature all the standard properties of bosonic Josephson junction arrays, including macroscopic quantum self-trapping. We focus on the harmonically trapped dipolar supersolids of interest for current experiments, and show that they can be described with a generalized Josephson model that takes into account spatial inhomogeneities. Our work shades new light on the dynamics of supersolids and opens the way to the study of a novel class of Josephson junctions.
## Dynamic Circuits for the Quantum Lattice-Boltzmann Method
- **Keywords:** ['hybrid quantum']
 - **Authors:** David Wawrzyniak, Josef Winter, Steffen Schmidt, Thomas Indiniger, Christian F. Janßen, Uwe Schramm, Nikolaus A. Adams
 - **Subjects:** Subjects:
Quantum Physics (quant-ph)
 - **Arxiv link:** https://arxiv.org/abs/arXiv:2502.02131
 - **Pdf link:** https://arxiv.org/pdf/arXiv:2502.02131
 - **Abstract**
 We propose a quantum algorithm for the linear advection-diffusion equation (ADE) Lattice-Boltzmann method (LBM) that leverages dynamic circuits. Dynamic quantum circuits allow for an optimized collision-operator quantum algorithm, introducing partial measurements as an integral step. Efficient adaptation of the quantum circuit during execution based on digital information obtained through mid-circuit measurements is achieved. The proposed new collision algorithm is implemented as a fully unitary operator, which facilitates the computation of multiple time steps without state reinitialization. Unlike previous quantum collision operators that rely on linear combinations of unitaries, the proposed algorithm does not exhibit a probabilistic failure rate. Moreover, additional qubits no longer depend on the chosen velocity set, which reduces both qubit overhead and circuit complexity. Validation of the quantum collision algorithm is performed by comparing results with digital LBM in one and two dimensions, demonstrating excellent agreement. Performance analysis for multiple time steps highlights advantages compared to previous methods. As an additional variant, a hybrid quantum-digital approach is proposed, which reduces the number of mid-circuit measurements, therefore improving the efficiency of the quantum collision algorithm.
## A mixed-precision quantum-classical algorithm for solving linear systems
- **Keywords:** ['hybrid quantum']
 - **Authors:** Océane Koska, Marc Baboulin, Arnaud Gazda
 - **Subjects:** Subjects:
Quantum Physics (quant-ph)
 - **Arxiv link:** https://arxiv.org/abs/arXiv:2502.02212
 - **Pdf link:** https://arxiv.org/pdf/arXiv:2502.02212
 - **Abstract**
 We address the problem of solving a system of linear equations via the Quantum Singular Value Transformation (QSVT). One drawback of the QSVT algorithm is that it requires huge quantum resources if we want to achieve an acceptable accuracy. To reduce the quantum cost, we propose a hybrid quantum-classical algorithm that improves the accuracy and reduces the cost of the QSVT by adding iterative refinement in mixed-precision A first quantum solution is computed using the QSVT, in low precision, and then refined in higher precision until we get a satisfactory accuracy. For this solver, we present an error and complexity analysis, and first experiments using the quantum software stack myQLM.
## Noise-Aware Mixed-State Quantum Computation via Parameterized Quantum Channels
- **Keywords:** ['hybrid quantum']
 - **Authors:** Giuseppe Clemente, Kevin Zambello
 - **Subjects:** Subjects:
Quantum Physics (quant-ph)
 - **Arxiv link:** https://arxiv.org/abs/arXiv:2502.02324
 - **Pdf link:** https://arxiv.org/pdf/arXiv:2502.02324
 - **Abstract**
 Non-unitary protocols are already at the base of many hybrid quantum computing applications, especially in the noisy intermediate-scale quantum (NISQ) era where quantum errors typically affect the unitary evolution. However, while the framework for Parameterized Quantum Circuits is widely developed, especially for applications where the parameters are optimized towards a set goal, we find there are still interesting opportunities in defining a unified framework also for non-unitary protocols in the form of Parameterized Quantum Channels as a computing resource. We first discuss the general parameterization strategies for controlling quantum channels and their practical realizations. Then we describe a simple example of application in the context of error mitigation, where the control parameters for the quantum channels are optimized in the presence of noise, in order to maximize channel fidelity with respect to a given target channel.
## Quantum Bayesian Inference with Renormalization for Gravitational Waves
- **Keywords:** ['hybrid quantum']
 - **Authors:** Gabriel Escrig, Roberto Campos, Hong Qi, M. A. Martin-Delgado
 - **Subjects:** Subjects:
Quantum Physics (quant-ph); High Energy Astrophysical Phenomena (astro-ph.HE); General Relativity and Quantum Cosmology (gr-qc)
 - **Arxiv link:** https://arxiv.org/abs/arXiv:2403.00846
 - **Pdf link:** https://arxiv.org/pdf/arXiv:2403.00846
 - **Abstract**
 Advancements in gravitational-wave interferometers, particularly the next generation, are poised to profoundly impact gravitational wave astronomy and multimessenger astrophysics. A hybrid quantum algorithm is proposed to carry out quantum inference of parameters from compact binary coalescences detected in gravitational-wave interferometers. It performs quantum Bayesian Inference with Renormalization and Downsampling (qBIRD). We choose binary black hole (BBH) mergers from LIGO observatories as the first case to test the algorithm, but its application can be extended to more general instances. The quantum algorithm is able to generate corner plots of relevant parameters such as chirp mass, mass ratio, spins, etc. by inference of simulated gravitational waves with known injected parameter values with zero noise, Gaussian noise and real data, thus recovering an accuracy equivalent to that of classical Markov Chain Monte Carlo inferences. The simulations are performed with sets of 2 and 4 parameters. These results enhance the possibilities to extend our capacity to track signals from coalescences over longer durations and at lower frequencies extending the accuracy and promptness of gravitational wave parameter estimation.
## Hybrid Quantum Neural Networks with Amplitude Encoding: Advancing Recovery Rate Predictions
- **Keywords:** ['hybrid quantum']
 - **Authors:** Ying Chen, Paul Griffin, Paolo Recchia, Lei Zhou, Hongrui Zhang
 - **Subjects:** Subjects:
Computational Finance (q-fin.CP); Machine Learning (cs.LG); Quantum Physics (quant-ph)
 - **Arxiv link:** https://arxiv.org/abs/arXiv:2501.15828
 - **Pdf link:** https://arxiv.org/pdf/arXiv:2501.15828
 - **Abstract**
 Recovery rate prediction plays a pivotal role in bond investment strategies, enhancing risk assessment, optimizing portfolio allocation, improving pricing accuracy, and supporting effective credit risk management. However, forecasting faces challenges like high-dimensional features, small sample sizes, and overfitting. We propose a hybrid Quantum Machine Learning model incorporating Parameterized Quantum Circuits (PQC) within a neural network framework. PQCs inherently preserve unitarity, avoiding computationally costly orthogonality constraints, while amplitude encoding enables exponential data compression, reducing qubit requirements logarithmically. Applied to a global dataset of 1,725 observations (1996-2023), our method achieved superior accuracy (RMSE 0.228) compared to classical neural networks (0.246) and quantum models with angle encoding (0.242), with efficient computation times. This work highlights the potential of hybrid quantum-classical architectures in advancing recovery rate forecasting.
